charts/xwiki/values.yaml

```diff
@@ -159,9 +159,27 @@ externalDB:
 ## To use external solr enable solr, provide host and port
 ##
 solr:
-  enabled: true
-  host: localhost
-  port: 8983
+  enabled: true #If set to false mount an internal Solr instance
```

Comment:
By default, let's assume false. Since the default use case is use internal solr from xwiki image. 

And will be its optional to use external solr. And required only for future version with support for cluster.

Reply:
You're right, it's better this way !